### ISSUE TITLE
Cannot run insect@4.0.3

### ISSUE THREAD

**Issue by setaur** [sharkdp/insect#65]
```
# npm install -g insect
npm WARN engine xdg-basedir@3.0.0: wanted: {"node":">=4"} (current: {"node":"0.10.29","npm":"1.4.21"})
/usr/local/bin/insect -> /usr/local/lib/node_modules/insect/index.js
insect@4.0.3 /usr/local/lib/node_modules/insect
├── xdg-basedir@3.0.0
├── line-reader@0.4.0
├── decimal.js@7.2.0
└── historic-readline@1.0.8 (fs-extra@0.24.0)
```
```
$ insect 

/usr/local/lib/node_modules/insect/node_modules/xdg-basedir/index.js:2
const os = require('os');
^^^^^
SyntaxError: Use of const in strict mode.
    at Module._compile (module.js:439:25)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/insect/index.js:50:20)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32
```
My os: Debian Jessie

**Comment by sharkdp** [comment]
Thanks for the feedback. I believe it should work with a more recent version of nodejs. I'm currently out on holidays but I will have a closer look when I'm back.

**Comment by sharkdp** [comment]
This should be fixed in [v4.2.0](https://github.com/sharkdp/insect/releases/tag/v4.2.0) of insect.

Could you please check if this works for you? Thank you very much.

**Comment by setaur** [comment]
Yeah, it's working great now.
